drivers/SmartThings/philips-hue/src/init.lua

```diff
@@ -705,8 +710,14 @@ local function do_bridge_network_init(driver, device, bridge_url, api_key)
 
       local bridge_api = device:get_field(Fields.BRIDGE_API)
       cosock.spawn(function()
+        -- Wait bit before doing querying to allow initial state for added devices to be reported
+        -- Children will all remain offline for this amount of time when a bridge comes back online.
+        -- Children will all be online for this amount of time when a bridge id first joined.
+        cosock.socket.sleep(15) --TODO is this long enough for large hue setup initial onboarding to complete for all devices?
```

Comment:
A delay is needed for the onboarding case where we do emit events for offline devices.  Perhaps there is an event based solution to do this scan at the opportune time, but I struggled to find an event for "all devices added" during the onboarding of a hue setup.